scripts/emerge/stage_prs_anvil_deliverable_data.py

```diff
@@ -82,9 +82,18 @@ def copy_object(src_bucket_name, src_object_name, dest_bucket_name, dest_object_
 
     source_bucket = storage_client.bucket(src_bucket_name, user_project=project_id)
     source_object = source_bucket.blob(src_object_name)
-    destination_bucket = storage_client.bucket(dest_bucket_name)
-
-    blob_copy = source_bucket.copy_blob(source_object, destination_bucket, dest_object_name)
+    destination_bucket = storage_client.bucket(dest_bucket_name, user_project=project_id)
+    destination_object = destination_bucket.blob(dest_object_name)
+
+    # rewrite instead of copy - https://cloud.google.com/storage/docs/json_api/v1/objects/copy
+    # TLDR; use rewrite vs copy: copy uses rewrite but only calls rewrite once.
+    # larger objects can require multiple rewrite calls leading to "Payload too large errors."    
```

Comment:
i didn't understand this fully at first, and read the documentation - i think it would be a little more clear to add to the end of the sentence `...leading to "Payload too large errors" when using copy; using rewrite in the following manner supports multiple rewrites`